src/main/java/org/jitsi/jigasi/transcription/GoogleCloudTranscriptionService.java

```diff
@@ -582,11 +563,8 @@ synchronized void sessionEnded()
             // round up to 15 second intervals
             int intervals15s = 1 + (int) (summedTime  / INTERVAL_LENGTH_MS);
 
-            if (client != null)
-            {
-                client.count(ASPECT_INTERVAL, intervals15s);
-                client.count(ASPECT_TOTAL_REQUESTS, requestsCount);
-            }
+            Statistics.incrementTotalTranscriber15sIntervals(intervals15s);
+            Statistics.incrementTotalTranscriberRequests();
```

Comment:
This one should probably be incremented directly in `incrementRequestsCounter`, or else be incremented *by* `requestsCount`

Reply:
Done.